index.src.html

```diff
@@ -137,8 +140,72 @@
+
+: <dfn>Authenticator</dfn>
+:: A cryptographic device used by a <a>[WAC]</a> to (i) generate a <a>scoped credential</a> and register it with a <a>[RP]</a>, and (ii) subsequently used to cryptographically sign and return, in the form of an <a>WebAuthn Assertion</a>, a challenge and other data presented by a <a>[RP]</a> (in concert with the <a>[WAC]</a>) in order to effect authentication. Broadly, authenticators are of two types: <a>Embedded Authenticators</a> or <a>External Authenticators</a>.
+
+: <dfn>Authorization Gesture</dfn>
+:: Essentially the same as <a>user verification</a>.
+
+: <dfn>Ceremony</dfn>
+:: The concept of a ceremony [[Ceremony]] is an extension of the concept of a network protocol, with human nodes alongside computer nodes and with communication links that include UI, human-to-human communication and transfers of physical objects that carry data. What is out-of-band to a  protocol is in-band to a ceremony. In this specification, <a>Registration</a>, <a>Authentication</a>, and <a>user verification</a> are ceremonies. 
+    
+: <dfn>Client</dfn>
+:: See <a>Conforming User Agent</a>.
+
+: <dfn>Conforming User Agent</dfn>
+:: A user agent implementing, in conjunction with the underlying platform, the <a>Web Authentication API</a> and algorithms given in this specification, and handling communication between 
+    <a>Authenticators</a> and <a>[RPS]</a>.
+
+: <dfn>eTLD+1</dfn>
+:: Also known as a <em>Registered Domain</em> [[PSL]], an eTLD+1 is an <em>effective Top-Level Domain Name</em> (eTLD), plus the next domain name label, proceding from right to left. An eTLD is also known as a <em>public suffix</em> [[RFC7719]].
+
+: <dfn>Registration</dfn>
+:: The <a>ceremony</a> where a user, a <a>[RP]</a>, and the user's computing device(s) (containing at least one <a>authenticator</a>) work in concert to create a <a>scoped credential</a> and associate it with the user's <a>[RP]</a> account.  Note that this includes employing <a>user verification</a>.
+
+: <dfn>Relying Party Identifier</dfn>
+: <dfn>RP ID</dfn>
+:: A Relying Party Identifier is derived from a <a>[RP]</a>'s web origin's hostname by computing the hostname's <a>eTLD+1</a>. 
+    
+: <dfn>Scoped Credential</dfn>
+:: Generically, a credential is data one entity presents to another in order to authenticate the former's identity [[RFC4949]]. A <em><a>scoped credential</a></em>, as defined here, is one that is cryptographically bound to a <a>[RP]</a>'s <a>RP ID</a>. Per specification, it is conveyed--i.e., registered--to only the <a>[RP]</a> instigating the credential's creation (see: <a>Registration</a>). Subsequently, only that <a>[RP]</a>, as identified by it's <a>RP ID</a> is able to employ the <a>scoped credential</a> in <a>authentication</a> ceremonies. 
```

Comment:
see "Tutorial" section of the "$ credential" entry in https://tools.ietf.org/html/rfc4949, especially last parag...
```
   $ credential
    [...]
      Tutorial: In general English, "credentials" are evidence or
      testimonials that (a) support a claim of identity or authorization
      and (b) usually are intended to be used more than once (i.e., a
      credential's life is long compared to the time needed for one
      use). [...]

      The problem with adopting this term in Internet security is that
      an automated process for authentication or access control usually
      requires multiple steps using multiple data objects, and it might
      not be immediately obvious which of those objects should get the
      name "credential".

      For example, if the verification step in a user authentication
      process employs public-key technology, then the process involves
      at least three data items: (a) the user's private key, (b) a
      signed value -- signed with that private key and passed to the
      system, perhaps in response to a challenge from the system -- and
      (c) the user's public-key certificate, which is validated by the
      system and provides the public key needed to verify the signature.
      -  Private key: The private key is *not* a credential, because it
         is never transferred or presented. Instead, the private key is
         "authentication information", which is associated with the
         user's identifier for a specified period of time and can be
         used in multiple authentications during that time.
      -  Signed value: The signed value is *not* a credential; the
         signed value is only ephemeral, not long lasting. The OSIRM
         definition could be interpreted to call the signed value a
         credential, but that would conflict with general English.
      -  Certificate: The user's certificate *is* a credential. It can
         be "transferred" or "presented" to any person or process that
         needs it at any time. A public-key certificate may be used as
         an "identity credential", and an attribute certificate may be
         used as an "authorization credential".
```